Source/Meadow.Foundation.Peripherals/Sensors.Hid.Xpt2046/Samples/Tsc2004_Sample/MeadowApp.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Meadow;
+using Meadow.Devices;
+using Meadow.Foundation.Graphics;
+using Meadow.Foundation.Sensors.Hid;
+using Meadow.Hardware;
+using Meadow.Peripherals.Displays;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Bbq10Keyboard_Sample
```

Comment:
I'm assuming this namespace is wrong in the Tsc2004 sample too

Reply:
LOL, I copied the sample project over, but never added it to the solution of actually updated it.  Doing that now.